Pod/Classes/Controller/BSImagePickerViewController.swift

```diff
@@ -64,11 +64,15 @@ open class BSImagePickerViewController : UINavigationController {
         return [cameraRollResult, albumResult]
     }()
     
-    var albumTitleView: AlbumTitleView = bundle.loadNibNamed("AlbumTitleView", owner: nil, options: nil)!.first as! AlbumTitleView
+    @objc var albumTitleView: UIButton = {
+        let btn =  UIButton(frame: .zero)
```

Comment:
I hope they call `sizeToFit` after setting the title because I've seen buttons continue to have frame `.zero` on iOS 10 and 9 🤞